manifests/config.pp

```diff
@@ -21,6 +21,16 @@
     target => $pulp::ca_cert,
   }
 
+  unless $pulp::manage_httpd {
```

Comment:
That complicates matters. We use the apache_version from the apache module. Won't that module try to manage apache and interfere? At the very least I think a short comment describing why it is intentional is in order.